- Ubuntu Sanity
I have fixed the pylint error.

- Ubuntu CC
I have no idea about this timeout, please provide the detail error log.
```
TIMEOUT: //tensorflow/core:lib_io_record_reader_writer_test (Summary)
      /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/core/lib_io_record_reader_writer_test/test.log
      /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/core/lib_io_record_reader_writer_test/test.log
      /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/core/lib_io_record_reader_writer_test/test_attempts/attempt_1.log
      /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/core/lib_io_record_reader_writer_test/test.log
      /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/core/lib_io_record_reader_writer_test/test_attempts/attempt_2.log
```

---

Those error seems not related to this PR:
- Linux GPU
`[  FAILED  ] IndexedCaseTest.testParallelExecution`

- MacOS Contrib
`tensorflow/lite/delegates/nnapi/nnapi_delegate.cc:764:36: error: use of undeclared identifier 'PROT_READ'`

- Windows Bazel & Windows Bazel GPU
`tensorflow/lite/mmap_allocation_disabled.cc(24): error C2065: 'kMMap': undeclared identifier`

--- a/tensorflow/core/lib/io/zlib_outputbuffer.cc
+++ b/tensorflow/core/lib/io/zlib_outputbuffer.cc
@@ -191,7 +191,7 @@ Status ZlibOutputBuffer::Append(StringPiece data) {
 }
 
 Status ZlibOutputBuffer::Flush() {
-  TF_RETURN_IF_ERROR(DeflateBuffered(Z_FULL_FLUSH));
+  TF_RETURN_IF_ERROR(DeflateBuffered(Z_PARTIAL_FLUSH));
   TF_RETURN_IF_ERROR(FlushOutputBufferToFile());
   return file_->Flush();
 }

